src/front/js/component/Navbar.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import "../../styles/navbar.css";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleUser } from '@fortawesome/free-solid-svg-icons';
+import { faCartShopping } from '@fortawesome/free-solid-svg-icons';
+import { faBars } from '@fortawesome/free-solid-svg-icons';
+import logoKusi from '../../img/Kusi.png'
+import { Link } from "react-router-dom";
+
+export const Navbar = () => {
+
+    const [ role, setRole ] = useState("chef")
+    const [isActive, setIsActive ] = useState(0)
```

Comment:
After reading your code, this value stores the index of the active tab, so it is not a boolean. You might consider renaming this state.